.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 script:
   - export PYTHONPATH=`pwd`:$PYTHONPATH
   - python src/tests/runtests.py
-  # - nunit-console src/embed_tests/bin/x64/ReleaseMono/Python.EmbeddingTest.dll
+  # - nunit-console src/embed_tests/bin/Python.EmbeddingTest.dll
```

Comment:
It runs the tests but all say `Cant find python.runtime` error. Didn't have a chance to debug it. I'm keeping the tests paths updated though